net/amneziawg-tools/Makefile

```
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2016-2019 Jason A. Donenfeld <Jason@zx2c4.com>
+# Copyright (C) 2016 Baptiste Jonglez <openwrt@bitsofnetworks.org>
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=amneziawg-tools
+PKG_VERSION:=1.0.20240213
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

This can not be accepted as it is. The PKG_SOURCE should always include PKG_NAME together with the PKG_VERSION. If you include only PKG_VERSION, this leads that you will download file: v1.0.20240213.tar.gz into the dl folder within OpenWrt build system. This file could be overwritten by other packages, which could have the same name (in this case version), but also if many projects will do that, then you will not know which file belongs to which files.

For example, how it should look with using codeload:
```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/amnezia-vpn/amneziawg-tools/tar.gz/v$(PKG_VERSION)?
```